mlkem/native/x86_64/src/basemul.c

```diff
@@ -8,15 +8,56 @@
 #if defined(MLKEM_NATIVE_ARITH_BACKEND_X86_64_DEFAULT)
 
 #include "consts.h"
+
 #include "poly.h"
 #include "polyvec.h"
 
 #include "arith_native_x86_64.h"
 
-static void poly_basemul_montgomery_avx2(poly *r, const poly *a, const poly *b)
+static const int16_t zetas_avx2[64] = {
+#include "x86_64_zetas_mulcache.i"
```

Comment:
Nit: Unless there's a good reason, I'd prefer the autogenerated files to be `.h` files. The other autogenerated AVX2 file isn't because it's included in the middle of an array, but here you could just generate the whole array in `x64_64_zetas_mulcache.h`.

Reply:
done